README.md

```diff
@@ -70,30 +82,28 @@ Requirements:
 - [Git](https://git-scm.com/downloads)
 
 **Install dependencies**
+
 ```shell
 yarn
 ```
+
 **Start server without building it** on http://localhost:8080/
+
 ```shell
 yarn start
 ```
-## Self-hosting
-
-Currently, only development version is available
 
-## Contribution
-
-Pull requests are welcome. For major changes, please open an
-[issue](https://github.com/InnoSWP/syntax-tree-visualization/issues/new) first
-to discuss what you would like to change.
-
-## Notable used tools
+## Technologies
```

Comment:
Why not Notable used tools?
Example of large project with the same heading https://github.com/excalidraw/excalidraw